[2024-04-29T15:38:01.988+0700] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-29T15:38:02.015+0700] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: DAG_Code-Competance-2.execute_load_to_warehouse manual__2024-04-29T08:30:05.304457+00:00 [queued]>
[2024-04-29T15:38:02.026+0700] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: DAG_Code-Competance-2.execute_load_to_warehouse manual__2024-04-29T08:30:05.304457+00:00 [queued]>
[2024-04-29T15:38:02.027+0700] {taskinstance.py:2303} INFO - Starting attempt 3 of 4
[2024-04-29T15:38:02.063+0700] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): execute_load_to_warehouse> on 2024-04-29 08:30:05.304457+00:00
[2024-04-29T15:38:02.069+0700] {standard_task_runner.py:63} INFO - Started process 128014 to run task
[2024-04-29T15:38:02.073+0700] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'DAG_Code-Competance-2', 'execute_load_to_warehouse', 'manual__2024-04-29T08:30:05.304457+00:00', '--job-id', '82', '--raw', '--subdir', 'DAGS_FOLDER/DAG_ETL_Code-Competance-2.py', '--cfg-path', '/tmp/tmphsdtr5yc']
[2024-04-29T15:38:02.075+0700] {standard_task_runner.py:91} INFO - Job 82: Subtask execute_load_to_warehouse
[2024-04-29T15:38:02.131+0700] {task_command.py:426} INFO - Running <TaskInstance: DAG_Code-Competance-2.execute_load_to_warehouse manual__2024-04-29T08:30:05.304457+00:00 [running]> on host LAPTOP-984VLCTV.
[2024-04-29T15:38:02.206+0700] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='nuri' AIRFLOW_CTX_DAG_ID='DAG_Code-Competance-2' AIRFLOW_CTX_TASK_ID='execute_load_to_warehouse' AIRFLOW_CTX_EXECUTION_DATE='2024-04-29T08:30:05.304457+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-29T08:30:05.304457+00:00'
[2024-04-29T15:38:02.208+0700] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-29T15:38:02.280+0700] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-29T15:38:02.281+0700] {taskinstance.py:2890} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/newrey/airflow/airflow-env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/newrey/airflow/airflow-env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/newrey/airflow/airflow-env/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
  File "/home/newrey/airflow/airflow-env/lib/python3.10/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
  File "/home/newrey/airflow/airflow-env/lib/python3.10/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/home/newrey/airflow/dags/DAG_ETL_Code-Competance-2.py", line 51, in execute_load_to_warehouse
    tickets_sold, revenue, avg_rating = loader.transform_data(transactions_df, tickets_df, events_df, customer_feedback_df)
NameError: name 'transactions_df' is not defined
[2024-04-29T15:38:02.285+0700] {taskinstance.py:1205} INFO - Marking task as UP_FOR_RETRY. dag_id=DAG_Code-Competance-2, task_id=execute_load_to_warehouse, execution_date=20240429T083005, start_date=20240429T083802, end_date=20240429T083802
[2024-04-29T15:38:02.301+0700] {standard_task_runner.py:110} ERROR - Failed to execute job 82 for task execute_load_to_warehouse (name 'transactions_df' is not defined; 128014)
[2024-04-29T15:38:02.329+0700] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-04-29T15:38:02.346+0700] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-29T15:38:02.347+0700] {local_task_job_runner.py:222} INFO - ::endgroup::
